reference/config_files/conan.conf.rst

```diff
@@ -218,6 +219,13 @@ On Windows:
     If you want to change the default "conan home" (directory where ``conan.conf`` file is) you can adjust
     the environment variable ``CONAN_USER_HOME``.
 
+
+The ``storage.download_cache`` variable defines the path to a folder that can be used to cache the different file downloads from Conan servers but
```

Comment:
is there also an environment variable for this?

Reply:
Nop. I want to get rid of env-var configuration and use conan.conf only configuration, to avoid env-vars being globally misused around the codebase and even to permeate to user recipes.